app/src/main/java/com/example/androidseminar/activity/SignInActivity.kt

```diff
@@ -84,13 +88,12 @@ class SignInActivity : BaseActivity<ActivitySignInBinding>({ ActivitySignInBindi
                 response: Response<ResponseWrapper<ResponseLoginData>>
             ) {
                 if(response.isSuccessful){
-                    val data=response.body()?.data
 
-                    //Toast.makeText(this@SignInActivity,"${data?.name}님 반갑습니다!",Toast.LENGTH_LONG).show()
+                    val data=response.body()?.data
                     shortToast("${data?.name}님 반갑습니다!")
```

Comment:
data가 null일 수도 있으니까 엘비스 연산자 써서 처리해주면 조금 더 깔끔할 것 같아
```suggestion
                    val data=response.body()?.data ?: "-"
                    shortToast("${data.name}님 반갑습니다!")
```